mission-control/docs/registry/kubernetes.md

```diff
@@ -71,4 +72,25 @@ The following table lists the configurable parameters and their default values:
 | `scraper.retention.changes` | Retention changes. | [{"name": "ReconciliationSucceeded", "count": 10}] |
 | `scraper.relationships` | Kubernetes relationships to create via name, namespace and kind. [Reference Docs](/config-db/scrapers/kubernetes#kubernetesrelationships) | [{"name": {"label": "helm.toolkit.fluxcd.io/name"}, {"name": {"label": "helm.toolkit.fluxcd.io/namespace"}, {"kind": {"value": "HelmRelease"}] |
 
-Feel free to customize this template according to your specific needs.
+
+### Playbooks
```

Comment:
Can we extract details into a tutorial and link it from here?

I think we also need a description on how to setup kustomize